aries-site/src/components/cards/ContentCard.js

```diff
@@ -21,21 +22,28 @@ export const ContentCard = forwardRef(({ topic, ...rest }, ref) => {
       {...rest}
     >
       <Box gap="large">
-        <Box
-          background="background-back"
-          height="small"
-          round="xsmall"
-          overflow="hidden"
-          style={{ position: 'relative' }}
-        >
-          {previewImage && (
-            <Image
-              src={previewImage.src}
-              alt={previewImage.alt}
-              fit={previewImage.fit || 'cover'}
-            />
-          )}
-        </Box>
+        {preview && (
+          <PreviewImageCard background={preview.background}>
+            {preview &&
+              (preview.image ? (
+                <Image
+                  src={preview.image.src}
+                  alt={preview.image.alt}
+                  fit={preview.image.fit || 'cover'}
+                />
+              ) : (
+                <Box
+                  style={{ pointerEvents: 'none' }}
+                  flex
+                  justify={preview.justify || 'center'}
+                  align="center"
+                >
+                  {preview.component()}
+                </Box>
+              ))}
+          </PreviewImageCard>
+        )}
+        {!preview && <PreviewImageCard />}
```

Comment:
Do we have to have this check? Wouldn't the condition on 27 just have undefined if there was no preview?

Reply:
unless Im missing something yeah otherwise it would put another card underneath even the ones with the preview ? unless you meant another way